contracts/access/roles/MinterRole.sol

```diff
@@ -0,0 +1,35 @@
+pragma solidity ^0.4.24;
+
+import "../Roles.sol";
+
+
+contract MinterRole {
+  using Roles for Roles.Role;
+
+  Roles.Role private minters;
+
+  constructor() public {
+    minters.add(msg.sender);
+  }
+
+  modifier onlyMinter() {
```

Comment:
I probably should've gone into some more detail in the PR description: the role contracts (everything under `access/roles/`, and associated tests) will be auto-generated in the future (see #1290), since they are all identical (we call it the 'role boilerplate'). This is sort of a limitation of how libraries work in Solidty, but I'm not yet convinced what'd be the best way to tackle this moving forward. For now, code repetition is what we have.